tests/test_pipelines.py

```diff
@@ -9,31 +9,53 @@
 
 from __future__ import absolute_import, print_function, unicode_literals
 
+import os
+
+import pkg_resources
+
 import pytest
 
+from scrapy.http import TextResponse
+
 from hepcrawl.spiders import aps_spider
 from hepcrawl.pipelines import InspireAPIPushPipeline, JsonWriterPipeline
 
 from .responses import fake_response_from_file
 
 
 @pytest.fixture
-def json_spider_record(tmpdir):
-    from scrapy.http import TextResponse
+def aps_xml_record():
+    """Returns the XML file where references will be scraped."""
+    return pkg_resources.resource_string(
+        __name__,
+        os.path.join(
+            'responses',
+            'aps',
+            'aps_single_response_test_ref.xml'
+        )
+    )
+
+@pytest.fixture
+def json_spider_record(tmpdir, aps_xml_record):
     spider = aps_spider.APSSpider()
-    items = spider.parse(fake_response_from_file('aps/aps_single_response.json', response_type=TextResponse))
+    request = spider.parse(
+        fake_response_from_file('aps/aps_single_response.json')
+    ).next()
+    response = TextResponse(
+        url=request.url,
+        request=request,
+        body=aps_xml_record,
+        **{'encoding': 'utf-8'}
```

Comment:
Jan did it like this, and I didn't want to break anything by changing his stuff.